cesium/internal/unary/writer.go

```diff
@@ -233,10 +239,11 @@ func (w *Writer) commitWithEnd(ctx context.Context, end telem.TimeStamp) (telem.
 	if !ok {
 		return 0, controller.Unauthorized(w.control.Subject.String(), w.Channel.Key)
 	}
+
 	if end.IsZero() {
 		// We're using w.len - 1 here because we want the timestamp of the last
 		// written frame.
-		approx, err := w.idx.Stamp(ctx, w.Start, w.len(dw.Writer)-1, true)
+		approx, err := w.idx.Stamp(ctx, w.Start, w.len(dw.Writer)-1, false)
```

Comment:
Why did we change this to false?

Reply:
now we must stamp across domains if the underlying "domain" was artificially split in two by file cutoff